src/Calendar.php

```diff
@@ -364,4 +364,33 @@ function (Udb3ModelOpeningHour $openingHour) {
 
         return new self($type, $startDate, $endDate, $timestamps, $openingHours);
     }
+
+    /**
+     * If the calendar has subEvents (timestamps), and a status manually set through an import or full calendar update
+     * through the API, the top status might be incorrect.
+     * For example the top status can not be Available if one of the subEvents is Unavailable.
```

Comment:
I think the top status can be `Available` even if one sub event is `Unavailable`. Because the top status is `Available` when a single sub event is `Available`. So maybe a better example is: the top status can not be `Available` if none of the sub events is `Available`?

Reply:
Ah yes the example in the comment is incorrect. I'll change it to your suggestion!